### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by SeraphWedd**
- Modified code to enable restart.
- Fixed minor syntax errors

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,157 +1,215 @@
...
+bluecoin = pygame.image.load('Images/bluecoin.jpg').convert()
+bomb = pygame.image.load('Images/bomb.png').convert()
 R = random.randint(1, 4)
-BG = pygame.image.load('Images/coinfallbg'+str(R)+'.jpg')
+BG = pygame.image.load('Images/coinfallbg'+str(R)+'.jpg').convert()
+
+cart_img.fill((255, 0, 0))
+coin_img.fill((255, 255, 0))
+bluecoin.fill((0, 0, 255))
+bomb.fill((125, 125, 125))
```
Comment (vineetjc) [inline main.py]: Running the game with the changes shows these colours instead of the images, please check

File: main.py [hunk 2]
```diff
@@ -1,157 +1,215 @@
...
-            B.draw(windowSurface)
-            B.fall()
-            cart.collect_item(B)
-        pygame.display.flip()
-        i+=1
-
-        #update time
+            #Add a restart key
+            elif event.type == KEYUP:
+                if event.key == pygame.K_r and (cart.dead or over):
```
Comment (vineetjc) [inline main.py]: Once the game ends, the timer restarts and still works even though 'R' isn't pressed. Please check.

**Comment by SeraphWedd** [comment]
I tried updating it.

The color fill is just junk code left after I tested the game without images.

The speed of the game probably increased due to the tick() so I restrained it to 30 fps.

The timer continuing even after game over is an unfiltered event. I added a simple fix.

**Comment by vineetjc** [comment]
@SeraphWedd well done on providing some fixes. I do have some feedback for you:

- Please be more accurate in naming the PRs wherever you're sending one. Instead of 'Add files via upload', maybe you could have named it something along the lines of 'Enable restart, fixed syntax' and so on. 

- I probably already mentioned about how some of your refactoring was nice. Well done on that.

- I may not fix all naming conventions after I do some clean up, but let this be for both of us: functions are all lower case and words are separated by an underscore. Let's be more accurate in Python's naming conventions.

I don't have much otherwise. Well done! Merging your PR.

**Comment by vineetjc** [comment]
@SeraphWedd I found some things while cleaning up:

- Some whitespace creeped in while you were coding. When you're sending PRs make sure they're removed.
- Some unnecessary else, else-pass statements which aren't needed per se.

Well done regardless!

**Pull request merged**